src/test/java/seedu/address/logic/commands/StartCommandTest.java

```diff
@@ -0,0 +1,40 @@
+package seedu.address.logic.commands;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.fail;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.logic.parser.AddressBookParser;
+import seedu.address.model.Model;
+import seedu.address.model.ModelManager;
+
+//@@author keiteo
+public class StartCommandTest {
+
+    private AddressBookParser addressBookParser = new AddressBookParser();
+    private Model model = new ModelManager();
```

Comment:
I think you should use a stub for this. There's one in AddCommandTest - you could refactor it out and reuse it.